Tasks/WindowsMachineFileCopy/WindowsMachineFileCopy.ps1

```diff
@@ -1,122 +1,104 @@
-param (
-    [string]$environmentName,
-    [string]$adminUserName,
-    [string]$adminPassword,
-    [string]$resourceFilteringMethod,
-    [string]$machineNames,
-    [string]$sourcePath,
-    [string]$targetPath,
-    [string]$additionalArguments,
-    [string]$cleanTargetBeforeCopy,
-    [string]$copyFilesInParallel
-    )
-
-Write-Verbose "Entering script WindowsMachineFileCopy.ps1"
-Write-Verbose "environmentName = $environmentName"
-Write-Verbose "adminUserName = $adminUserName"
-Write-Verbose "resourceFilteringMethod = $resourceFilteringMethod"
-Write-Verbose "machineNames = $machineNames"
-Write-Verbose "sourcePath = $sourcePath"
-Write-Verbose "targetPath = $targetPath"
-Write-Verbose "additionalArguments = $additionalArguments"
-Write-Verbose "copyFilesInParallel = $copyFilesInParallel"
-Write-Verbose "cleanTargetBeforeCopy = $cleanTargetBeforeCopy"
+[CmdletBinding()]
+param()
 
-. $PSScriptRoot/RoboCopyJob.ps1
-. $PSScriptRoot/Utility.ps1
-
-import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
-import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
-import-module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
-import-module "Microsoft.TeamFoundation.DistributedTask.Task.Deployment.Internal" -ErrorAction Ignore
-
-# keep machineNames parameter name unchanged due to back compatibility
-$machineFilter = $machineNames
-$sourcePath = $sourcePath.Trim('"')
-$targetPath = $targetPath.Trim('"')
+Trace-VstsEnteringInvocation $MyInvocation
 
-# Default + constants #
-$resourceFQDNKeyName = Get-ResourceFQDNTagKey
+# Get inputs for the task
+$machineNames = Get-VstsInput -Name MachineNames
+$adminUserName = Get-VstsInput -Name AdminUserName
```

Comment:
Should we make username/password mandatory now? And should we check if the format is proper for user name?